spec/lib/import/csv_parser_sections_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+require 'import/csv_parser'
+
+describe Import::CSVParser do
+  let(:input_file) { File.join(fixture_path, 'csv', 'import_sections', 'tempest_sections.csv') }
+  let(:parser) { described_class.new(input_file) }
+
+  before do
+    # Don't print status messages during specs
+    allow($stdout).to receive(:puts)
+  end
+
+  describe 'initializer' do
+    it 'has an input file' do
+      expect(parser.file).to eq input_file
+    end
+  end
+
+  describe '#attributes' do
```

Comment:
I think we can just combine this test with the similar test here:
https://github.com/curationexperts/heliotrope/blob/master/spec/lib/import/csv_parser_spec.rb#L22

And then this whole spec file can be deleted.